runtime/src/bank_forks.rs

```diff
@@ -694,6 +695,16 @@ impl BankForks {
             && bank.parent_slot() < start_slot
             && bank.slot() >= start_slot
     }
+
+    pub fn new_bank_from_parent_for_tests(
```

Comment:
Since `new_with_fork_for_tests()` returns the instance of `BankForks`, we can insert the new bank to forks in the tests itself. It's just two lines of code at the various call sites.

Reply:
I know two lines does not sound a lot, but function this called 57 times after fixing all the tests in the `runtime` folder. It facilitates writing tests.

If not as a method of `BankForks` I can code it as standalone function or add the `#[cfg(tests)]` attribute. Is there any chance I can keep the function?

Reply:
How about move it to `tests.rs` as a utility function? Keeping it it `bank_forks` increases the possibility of accidental misuse.